test/tests/functional/pbs_cray_vnode_per_numa.py

```diff
@@ -128,6 +128,31 @@ def test_settings(self):
                 self.assertEqual(int(n['resources_available.naccelerators']),
                                  dacc[n['resources_available.host']])
 
+        # Set vnode_per_numa_node to FALSE and re-read the vnodes
+        rv = self.mom.add_config({'$vnode_per_numa_node': False}, False)
+        self.assertTrue(rv)
+        self.reset_nodes(momname)
+
+        vnodes_combined1 = self.server.status(NODE)
+
+        # Compare the pbsnodes output when vnode_per_numa_node was unset
+        # versus when vnode_per_numa_node was set to False.
+
+        if (len(vnodes_combined) == len(vnodes_combined1)):
+            self.logger.info(
+                "pbsnodes outputs are equal in length.")
+            for n in vnodes_combined:
+                if n not in vnodes_combined1:
+                    self.logger.error(
+                        "ERROR vnode %s has differing element." % n['id'])
+                    self.assertTrue(False)
```

Comment:
These self.assertTrue(False) statements get the job done, but they seem awkward to me.  Is there a better way to do this?